lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -536,6 +536,11 @@ void CommandInterpreter::Initialize() {
   if (cmd_obj_sp) {
     AddAlias("h", cmd_obj_sp);
   }
+
+  cmd_obj_sp = GetCommandSPExact("target switch");
+  if (cmd_obj_sp) {
+    AddAlias("ts", cmd_obj_sp);
```

Comment:
Do we need this short alias? Or should we let users create it? Wondering if it might conflict with "target select" aliases in the wild.